strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/TempRepositoryPath.java

```diff
@@ -24,7 +24,7 @@ private TempRepositoryPath(RepositoryPath tempPath)
         super(tempPath.getTarget(), tempPath.getFileSystem());
     }
 
-    protected RepositoryPath getTempTarget()
+    public RepositoryPath getTempTarget()
```

Comment:
why this need to be `public`?

Reply:
not needed anymore, thanks for spotting this :)

Reply:
No, I still need it sorry. See https://github.com/strongbox/strongbox/pull/754/files#diff-fbc35a1e7a3762e40f94a3403ae94151R49